README.md

```diff
@@ -1 +1,46 @@
-# HelloMicroverseProject
\ No newline at end of file
+# HelloMicroverseProject
+
+- This is the first project at Microverse first week activity.
+- After creating the remote repository clone it using the command git clone git@github.com:aster-alemu/HelloMicroverseProject.git 
+- The next step was to make sure that there is no linter errors and for this I set-up gitHub actions and set-up linters in my local environment
+- Then create branch repository using the command git branch which is followed by the name of new branch
+- After creating the branch repository, switch from main branch to new branch using the command git checkout which is followed by the name of new branch created
+- Then create index.html file and write the contents
+- After that I create style.css file and link with the index.html file using the link tag
+- Then seening the result by opening the index.html file using Live Server
+- I check the if there is style errors by runing the command npx stylelint "**/*.{css,scss}"
+- I check if there is error on html file my runing the command npx hint . on the terminal
+- Then stage the all the files using the command git add . on the terminal
+- Then update the file using the command git commit -m followed by the commit message with double qouts on the terminal
+- Push the files using the command git push --set-upstream origin followed by the name of the branch on the terminal
```

Comment:
- [ ] Instead of having the project's description mixed in with the getting started section, kindly put them into two different sections. You can put the steps to clone and run the project in a new `Getting Started` section under the project's description.